conda/utils.py

```diff
@@ -98,22 +98,29 @@ def unix_path_to_win(path, root_prefix=""):
     if len(path) > 1 and (";" in path or (path[1] == ":" and path.count(":") == 1)):
         # already a windows path
         return path.replace("/", "\\")
-    """Convert a path or :-separated string of paths into a Windows representation"""
-    path_re = root_prefix +'(/[a-zA-Z]\/(?:[^:*?"<>|]+\/)*[^:*?"<>|;]*)'
-    translation = lambda found_path: found_path.group(0)[len(root_prefix)+1] + ":" + \
-                  found_path.group(0)[len(root_prefix)+2:].replace("/", "\\")
-    translation = re.sub(path_re, translation, path)
-    translation = re.sub(":?([a-zA-Z]):\\\\", lambda match: ";" + match.group(0)[1] + ":\\", translation)
+
+    path_re = root_prefix + '(/[a-zA-Z]\/(?:[^:*?"<>|]+\/)*[^:*?"<>|;]*)'
+
+    def _translation(found_path):
+        group = found_path.group(0)
+        return group[len(root_prefix) + 1] + ":" + group[len(root_prefix) + 2:].replace("/", "\\")
```

Comment:
Here, just because this PR is all about style, I think this is clearer:

    return "{0}:{1}".format(group[len(root_prefix) + 1],
                            group[len(root_prefix) + 2:].replace("/", "\\"))

And now that I'm looking at this, is the style around the `+` signs actually correct?  I'm guessing it is, but here I would have myself preferred

    return "{0}:{1}".format(group[len(root_prefix)+1],
                            group[len(root_prefix)+2:].replace("/", "\\"))

`¯\_(ツ)_/¯`